connectors/slack/template/storage.js

```diff
@@ -0,0 +1,48 @@
+const Sdk = require('@fusebit/add-on-sdk');
+
+let storage;
+module.exports.ensureStorage = (ctx) => {
+  if (!storage) {
+    storage = Sdk.getStorageClient(ctx);
+  }
+};
+
+module.exports.teamToHandler = null;
+module.exports.tokenToTeam = null;
```

Comment:
What's the purpose of setting to null? Can't they just be undefined?

Reply:
Fixed